packages/server/graphql/mutations/helpers/safelyWithdrawVote.ts

```diff
@@ -13,23 +15,40 @@ const safelyWithdrawVote = async (
 ) => {
   const meetingMemberId = toTeamMemberId(meetingId, userId)
   const r = await getRethink()
+  const pg = getKysely()
   const now = new Date()
   const viewerId = getUserId(authToken)
-  const isVoteRemovedFromGroup = await r
-    .table('RetroReflectionGroup')
-    .get(reflectionGroupId)
-    .update((group: RValue) => {
-      return r.branch(
-        group('voterIds').offsetsOf(userId).count().ge(1),
-        {
-          updatedAt: now,
-          voterIds: group('voterIds').deleteAt(group('voterIds').offsetsOf(userId).nth(0))
-        },
-        {}
-      )
-    })('replaced')
-    .eq(1)
-    .run()
+  const [isVoteRemovedFromGroup, voteRemovedResult] = await Promise.all([
+    r
+      .table('RetroReflectionGroup')
+      .get(reflectionGroupId)
+      .update((group: RValue) => {
+        return r.branch(
+          group('voterIds').offsetsOf(userId).count().ge(1),
+          {
+            updatedAt: now,
+            voterIds: group('voterIds').deleteAt(group('voterIds').offsetsOf(userId).nth(0))
+          },
+          {}
+        )
+      })('replaced')
+      .eq(1)
+      .run(),
+    pg
+      .updateTable('RetroReflectionGroup')
+      .set({
+        voterIds: sql`array_cat(
+          "voterIds"[1:array_position("voterIds",'google-oauth2|ohlApAJnXy')-1],
+          "voterIds"[array_position("voterIds",'google-oauth2|ohlApAJnXy')+1:]
+        )`
+      })
+      .where('id', '=', reflectionGroupId)
+      .where(sql`${userId}`, '=', sql`ANY("voterIds")`)
+      .executeTakeFirst()
+  ])
+  const isVoteRemovedFromGroupPG = voteRemovedResult.numUpdatedRows === BigInt(1)
+  if (isVoteRemovedFromGroup !== isVoteRemovedFromGroupPG)
+    console.log('MISMATCH VOTE REMOVED LOGIC')
```

Comment:
Should we log this kind of errors in sentry?

Reply:
yep! that's probably easier to get to than datadog